src/visual.ts

```diff
@@ -1294,7 +1300,9 @@ module powerbi.extensibility.visual {
 
                 if (visSettings.scrollAutoAdjustment.show && isSelected && !singleCaseDone) {
                     const selectedGranulaPos: number = (cellSelection[0][index] as any).x.baseVal.value;
-                    (this.mainSvgWrapperSelection[0][0] as any).scrollTo(selectedGranulaPos - this.horizontalAutoScrollingPositionOffset, 0);
+                    this.scrollAutoFocusFunc = () => {
```

Comment:
It'd be better not to create a new function for each `renderLabels` call.
The `scrollAutoFocusFunc` method must accept selectedGranulaPos as an argument